test/integration/host_js_test.rb

```diff
@@ -763,12 +739,21 @@ class HostJSTest < IntegrationTestWithJavascript
 
   private
 
+  def switch_form_tab_to_interfaces
+    switch_form_tab('Interfaces')
+    disable_interface_modal_animation
+  end
+
   def subnet_and_domain_are_selected(modal, domain)
```

Comment:
These were not `assert`ed, so not taken in account. this is less readable code though :( select2 ain't easy.